docs/plugins/composable_resources.md

```diff
@@ -169,6 +169,20 @@ select count(*) from (select data_id from r_data_main where resc_name = 'ROOT_OF
 
 These two queries can be added as specific queries via `iadmin asq`.
 
+#### Replication retry
+
+The replication resource can be configured to retry (or not) a replication upon failure via three settings in the context string. The retry mechanism also applies to replications in a Rebalance operation. Note: The retry will only occur on failure to *replicate* from one sibling to another. If the initial file transfer fails for some reason, the retry mechanism will not be initiated.
+
+The settings are configured like so (default values used):
+
+```
+context="retry_attempts=1;first_retry_delay_in_seconds=1;backoff_multiplier=1.0"
```

Comment:
Can this be an `iadmin modresc` command?  The 'context=' syntax is misleading here...?